CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # pupa changelog
 
+## 0.10.3 - April 3 2023
```

Comment:
How about bumping to 0.11.0, since this is a big change? We should also cut a release of python-opencivicdata, again bumping the version, and update our dep here.